release-notes/5.0/5.0-supported-os.md

```diff
@@ -45,32 +44,20 @@ Note: .NET 5 requires OpenSSL 1.x. Newer distro versions such as Ubuntu 22.04 us
 [Alpine-lifecycle]: https://wiki.alpinelinux.org/wiki/Alpine_Linux:Releases
 [CentOS]: https://www.centos.org/
 [CentOS-lifecycle]:https://wiki.centos.org/FAQ/General
```

Comment:
it seems this is the archived version?
```suggestion
[CentOS-lifecycle]:https://wiki.centos.org/FAQ(2f)General.html
```

Reply:
Yes, see https://github.com/dotnet/core/pull/9594